Formula/mupdf.rb

```diff
@@ -36,14 +22,6 @@ def install
     # Work around bug: https://bugs.ghostscript.com/show_bug.cgi?id=697842
     inreplace "Makerules", "RANLIB_CMD := xcrun", "RANLIB_CMD = xcrun"
```

Comment:
If we have this work-around, we could thus remove `HAVE_GLFW=no` below

Reply:
won't we have to put that back later?

Reply:
Don't think so.

Reply:
Upstream fixed the bug, actually, so we need to remove the `inreplace` too